apps/web/test/utils/bookingScenario/bookingScenario.ts

```diff
@@ -1164,18 +1190,36 @@ export function mockCalendar(
               log.silly("mockCalendar.updateEvent", JSON.stringify({ uid, event, externalCalendarId }));
               // eslint-disable-next-line prefer-rest-params
               updateEventCalls.push(rest);
-              return Promise.resolve({
-                type: app.type,
-                additionalInfo: {},
-                uid: "PROBABLY_UNUSED_UID",
-                iCalUID: normalizedCalendarData.update?.iCalUID,
-
-                // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-                id: normalizedCalendarData.update?.uid || "FALLBACK_MOCK_ID",
-                // Password and URL seems useless for CalendarService, plan to remove them if that's the case
-                password: "MOCK_PASSWORD",
-                url: "https://UNUSED_URL",
-              });
+              if (event.location === BookingLocations.GoogleMeet) {
+                return Promise.resolve({
+                  type: app.type,
+                  additionalInfo: {},
+                  uid: "PROBABLY_UNUSED_UID",
+                  iCalUID: normalizedCalendarData.update?.iCalUID,
+
+                  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+                  id: normalizedCalendarData.update?.uid || "FALLBACK_MOCK_ID",
+                  // Password and URL seems useless for CalendarService, plan to remove them if that's the case
+                  password: "MOCK_PASSWORD",
+                  url: "https://UNUSED_URL",
+                  location: "https://UNUSED_URL",
+                  hangoutLink: "https://UNUSED_URL",
+                  conferenceData: event.conferenceData,
```

Comment:
nit: We could just add these props conditionally.  Allows us to easily understand the difference